### ISSUE TITLE
Dashboard: Do not show site selector if there is only one site

### ISSUE THREAD

**Issue by tsteur** [piwik/piwik#8005]
Maybe even hide the "All Websites Dashboard" link.

**Comment by mnapoli** [comment]
Good idea to hide the "All Websites Dashboard", but I first started to use Piwik I was looking for "creating a new site" in the site manager (which created a lot of frustration, maybe now it's better with the "Administration" menu item). I think it might be good to add such a button there, so maybe we ought to keep it instead?

**Comment by tsteur** [comment]
Yeah we could do that. Many installations will always have only one installation though and in this case it shows "clutter" all the time. Maybe we could be worth to leave the "All Websites Dashboard" link but still remove the site selector. I'd personally remove both for simple UI but leaving the All Websites would be ok for me.

**Comment by mattab** [comment]
> I'd personally remove both for simple UI but leaving the All Websites would be ok for me.

Makes sense to remove the Site selector when there is one website,
but  imho may be better to leave the "All Websites" link at all times.

**Comment by AndersBillLinden** [comment]
We have 150 sites. Clicking the link "All Websites" results in a timeout after a while.

**Comment by tsteur** [comment]
That should be fixed in next release. If it is still slow with next version then most likely browser archiving is not disabled which should be the only reason that could cause it to be slow from then on.

**Comment by AndersBillLinden** [comment]
Can it be changed so that the list of servers it fetched instantly, rendering the page at once and then the extras on each line are loaded sequentially with ajax? Will make the page alive even if there is a lot of servers in the list.

**Comment by tsteur** [comment]
Do you mind creating a new issue for this? This issue is only about not showing the site selector

**Comment by AndersBillLinden** [comment]
:+1:

**Comment by mattab** [comment]
> Do not show site selector if there is only one site

don't really want to do this, because IMO there is value is showing the website name at all times in the UI, to clarify what we are looking at, in case we make screenshots, etc. I still leave issue opened in case other people want to comment.

**Comment by tsteur** [comment]
We could maybe instead of displaying a `select` remove the icon and the possibility to open it when there's only one website but that's maybe also not optimal since it would still look clickable but isn't. I reckon when someone only has one site then one knows on which site one is so I'd remove it... it adds a lot of clutter there. I reckon best thing would be to think about this again when working on the left menu.
